src/ReadStreamCSV.h

```diff
@@ -228,16 +228,17 @@ class ReadStreamCSV : public ReadStream {
 
 class ReadFileCSV : public ReadStreamCSV {
 public:
-    ReadFileCSV(const RWOperation& rwOperation, SymbolTable& symbolTable, RecordTable& recordTable)
+    ReadFileCSV(const std::map<std::string, std::string>& rwOperation, SymbolTable& symbolTable,
+            RecordTable& recordTable)
             : ReadStreamCSV(fileHandle, rwOperation, symbolTable, recordTable),
               baseName(souffle::baseName(getFileName(rwOperation))),
               fileHandle(getFileName(rwOperation), std::ios::in | std::ios::binary) {
-        if (!rwOperation.has("intermediate")) {
+        if (rwOperation.find("intermediate") == rwOperation.end()) {
```

Comment:
I think you can actually remove the if statement entirely and always execute the body. Loading of intermediate relations for incremental execution was removed a while ago.